common/views/components/DropdownButton/DropdownButton.tsx

```diff
@@ -142,9 +142,15 @@ const DropdownButton: FunctionComponent<PropsWithChildren<Props>> = ({
   });
 
   useEffect(() => {
-    dropdownRef &&
-      dropdownRef.current &&
-      setFocusables(getFocusableElements(dropdownRef.current));
+    if (dropdownRef && dropdownRef.current) {
+      const focusableElements = getFocusableElements(
+        dropdownRef.current
+      ) as HTMLElement[];
+      setFocusables(focusableElements);
+      focusableElements.forEach(focusable =>
```

Comment:
Wondering how this is different to https://github.com/wellcomecollection/wellcomecollection.org/blob/set-dropdown-items-to-unfocusable-on-mount/common/views/components/DropdownButton/DropdownButton.tsx#L160 - would it work to add `focusables` to the dependencies array there alongside `isActive` and `children`?